samples/typescript/package.json

```diff
@@ -8,10 +8,10 @@
     "test": "yarn tsc && node build/quickstart.js"
   },
   "dependencies": {
-    "@cognite/sdk": "^2",
-    "typescript": "^3.3.4000"
+    "@cognite/sdk": "^2"
```

Comment:
fyi we should bump this to v3

Reply:
We would have to do this after release. Normally lerna bumps package versions automatically but in this case we run `lerna version --no-private` which doesn't touch private packages. We made it that way because otherwise we'll get a bunch of "typescript-sample@2.3.1" git tags whenever stable changes.

So we have to update manually to x^ whenever major release happens, but otherwise it is fine, it uses local instance.